maas/data_source_maas_boot_source_test.go

```diff
@@ -9,15 +9,14 @@ import (
 )
 
 func TestAccDataSourceMaasBootSource_basic(t *testing.T) {
-
 	url := "http://images.maas.io/ephemeral-v3/stable/"
-	keyring_path := "/snap/maas/current/usr/share/keyrings/ubuntu-cloudimage-keyring.gpg"
+	keyringPath := "/snap/maas/current/usr/share/keyrings/ubuntu-cloudimage-keyring.gpg"
 
 	checks := []resource.TestCheckFunc{
 		resource.TestCheckResourceAttr("data.maas_boot_source.test", "url", url),
 		resource.TestCheckResourceAttrSet("data.maas_boot_source.test", "created"),
 		resource.TestCheckResourceAttr("data.maas_boot_source.test", "keyring_data", ""),
-		resource.TestCheckResourceAttr("data.maas_boot_source.test", "keyring_filename", keyring_path),
+		resource.TestCheckResourceAttr("data.maas_boot_source.test", "keyringFilename", keyringPath),
```

Comment:
Could it be a potential breaking change? (the string value change)

Reply:
this is a wrong change. `keyring_filename` is a property of a resource. Thank you for spotting it. I am reverting it now.